projects/discussion-ui/src/lib/components/discuss-all/discuss-all.component.scss

```diff
@@ -1,135 +1,24 @@
-// @import 'ws-common';
-// @import 'ws-vars';
-// @import 'ws-mixins';
-// @import "node_modules/bootstrap/scss/bootstrap";
```

Comment:
Why these CSS are removed ?

Reply:
these CSS are unused. I checked what class are there inside html that i kept. other unused css i removed.